platform/windows/display_server_windows.cpp

```diff
@@ -3382,11 +3414,20 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 				WindowRect.top,
 				WindowRect.right - WindowRect.left,
 				WindowRect.bottom - WindowRect.top,
-				nullptr, nullptr, hInstance, nullptr);
+#if 0
+				// set owner/parent for all windows except the main window
+				(id == MAIN_WINDOW_ID)?nullptr:windows[MAIN_WINDOW_ID].hWnd,
```

Comment:
Since this is always compiled out, is it safe to just remove from the PR?

Reply:
I put this here as a suggestion of sorts.  I added this code and then decided not to enable it, because I don't know if you would like it.   If you don't want to do that, then yes it could be removed.

Reply:
specifically, the utility is that Windows has API function that operate on "all WS_POPUP windows with owner set to the current window" and so I thought it would be useful to actually set the owner to the main window.  But since we aren't using those functions right now, I think it is up to you if you want it there.

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showownedpopups

Reply:
I see, I think it is fine to leave in for now with an update to the comment saying it would be necessary to add if we want to operate on all popups owned by a main window. 

I don't know if the other platforms have equivalent functionality or if it is something that would be useful (this isn't my area of expertise).

Reply:
Normally, parent is set only for the exclusive popups (it's handled in `window_set_exclusive`), there's no reason to set it here, and it will most likely break current behavior.

Reply:
will terminate with prejudice

Reply:
done